Sources/SWBUtil/FSProxy.swift

```diff
@@ -731,14 +731,28 @@ class LocalFS: FSProxy, @unchecked Sendable {
     }
 
     @discardableResult func traverse<T>(_ path: Path, _ f: (Path) throws -> T?) throws -> [T] {
-        guard let enumerator = fileManager.enumerator(atPath: path.str) else {
+        var error: (any Error)?
+        let errorHandler: (URL, any Error) -> Bool = { _, e in
+            error = e
+            return false
+        }
+
+        guard let enumerator = fileManager.enumerator(at: URL(fileURLWithPath: try realpath(path).str), includingPropertiesForKeys: nil, errorHandler: errorHandler) else {
```

Comment:
Why introduce `realpath` here?

Reply:
`FileManager.enumerator(at:includingPropertiesForKeys:options:errorHandler:)` is [a little different](https://developer.apple.com/documentation/foundation/filemanager/2765464-enumerator) from the path-based version:

> ### Parameters
> #### url
> The location of the directory for which you want an enumeration. This URL must not be a symbolic link that points to the desired directory. You can use the [resolvingSymlinksInPath](https://developer.apple.com/documentation/foundation/nsurl/1415965-resolvingsymlinksinpath) method to resolve any symlinks in the URL.

We could do that, or we can just call realpath to do it ourselves. Since our version plumbs errors through I thought it was a better choice.

Reply:
Darn. I think having to realpath or resolve symlinks introduces semantics changes and that might be a whole project to work through. Let's just go back to the previous implementation and track that as a tail task?

Reply:
Ok, I've reverted the change.